test/util/collectBedrockPackets.js

```diff
@@ -0,0 +1,52 @@
+const bedrock = require('bedrock-protocol')
```

Comment:
When should this be used ?
Can you add it to the doc ?

Reply:
The tests seem the same as the existing pc tests here https://github.com/CreeperG16/prismarine-registry/blob/master/test/util/collectMcpcPackets.js

Basically, start a server, connect a client to the server, test that client connects and prismarine-registry APIs work